### PR TITLE
 HEY-2653

### PR DESCRIPTION
**Pull request by AndreyShchagin**
AI service implemented and ready to be deployed

### CODE REVIEW

File: src/main/kotlin/car/hey/ai/model/dao/Contact.kt [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    var createdAt: Instant? = null
+        private set
+
+    var lastUpdatedAt: Instant? = null
+        private set
+
+    /**
+     * @param fullName the name of the company
+     */
+    var fullName: String
```
Comment (pecota) [inline src/main/kotlin/car/hey/ai/model/dao/Contact.kt]: This starts with a deprecated field?

File: src/main/resources/banner.txt [hunk 2]
```diff
@@ -1,37 +1,34 @@
-      _        _____      ______                               _
-     / \      |_   _|   .' ____ \                             (_)
-    / _ \       | |     | (___ \_|  .---.   _ .--.   _   __   __    .---.   .---.
-   / ___ \      | |      _.____`.  / /__\\ [ `/'`\] [ \ [  ] [  |  / /'`\] / /__\\
- _/ /   \ \_   _| |_    | \____) | | \__.,  | |      \ \/ /   | |  | \__.  | \__.,
-|____| |____| |_____|    \______.'  '.__.' [___]      \__/   [___] '.___.'  '.__.'
+                                             ____                                         ______    ____
```
Comment (pecota) [inline src/main/resources/banner.txt]: We need these banners?
On the future we could remove all.
That only create useless log lines on kibana.

File: src/test/kotlin/car/hey/ai/ws/DataScienceWsTest.kt [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.web.client.HttpServerErrorException
+import kotlin.test.assertNotNull
+
+class DataScienceWsTest {
+    @Rule
+    @JvmField
+    var dsWebServiceRule = WireMockRule(5000)
+
+    var dataScienceWs: DataScienceWs? = null
```
Comment (pecota) [inline src/test/kotlin/car/hey/ai/ws/DataScienceWsTest.kt]: Why not?    
`lateinit var dataScienceWs`

File: src/main/kotlin/car/hey/ai/model/dao/Contact.kt (hunk 1 above)
Comment (AndreyShchagin) [reply src/main/kotlin/car/hey/ai/model/dao/Contact.kt]: Yep this is 1 to 1 Lead DAO from leads service. Once the model will be finally migrated there we will have to do the same on AI service. Another reason to use common libraries for DTOs

File: src/main/resources/banner.txt (hunk 2 above)
Comment (AndreyShchagin) [reply src/main/resources/banner.txt]: Ok I will change it and keep it low ))

Comment (tobiastrinks) [reply src/main/resources/banner.txt]: Oh those banners are great! Ask @gregoriokusowski he can confirm. 😄

**Pull request merged**